lib/stripe/subscriptions/subscription.ex

```diff
@@ -137,7 +136,6 @@ defmodule Stripe.Subscription do
                optional(:metadata) => Stripe.Types.metadata(),
                optional(:prorate) => boolean,
                optional(:proration_date) => Stripe.timestamp(),
-               optional(:source) => Stripe.id() | Stripe.Source.t(),
                optional(:tax_percent) => float,
                optional(:trial_end) => Stripe.timestamp(),
                optional(:trial_from_plan) => boolean
```

Comment:
@mcrumm A couple more spots to remove `:source`, specifically in the `cast_to_id` helper function.

Reply:
@snewcomer Should be good to go now